package-lock.json

```diff
@@ -19196,6 +19196,7 @@
       "version": "1.11.5",
       "license": "MIT",
       "dependencies": {
+        "@grpc/grpc-js": "^1.12.4",
```

Comment:
That should no longer be here. Make sure to run `npm install` to update the lock file.

Reply:
Oops, 303cd122f5446d8715812c388e610436ad536dbe